### PR TITLE
Derivation fixes

### PR DESCRIPTION
**Pull request by Odomontois**

### CODE REVIEW

File: modules/kernel/src/main/scala-2.13/tschema/common/Name.scala [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package tschema.common
+
+final class Name[name](val string: String) {
+  def symbol: Symbol = Symbol(string)
```
Comment (ulanzetz) [inline modules/kernel/src/main/scala-2.13/tschema/common/Name.scala]: Is `symbol` used somewhere?

Comment (Odomontois) [reply modules/kernel/src/main/scala-2.13/tschema/common/Name.scala]: No, it's not. Good catch

**Comment by danslapman** [comment]
I don't think that namespace fragmentation is a good idea

**Comment by danslapman** [comment]
(I'm also against any namespace changes)

**Comment by Odomontois** [comment]
So revert everything back to ru.tinkoff.tschema?

**Comment by Odomontois** [comment]
> (I'm also against any namespace changes)

So revert everything back to ru.tinkoff.tschema?

**Comment by danslapman** [comment]
@Odomontois yeah, I propose so. I outlined my doubts here: https://github.com/TinkoffCreditSystems/typed-schema/pull/226
At least I think namespaces should be consistent

**Pull request merged**